doc/source/ray-core/examples/batch_training.ipynb

```diff
@@ -459,27 +447,35 @@
+      "[('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 145, [(DecisionTreeRegressor(), 618.3188066672511), (DecisionTreeRegressor(splitter='random'), 619.603168803982), (LinearRegression(), 848.0861910655393)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 161, [(DecisionTreeRegressor(), 578.1630239752242), (DecisionTreeRegressor(splitter='random'), 578.1630239752242), (LinearRegression(), 736.7727357111179)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 163, [(DecisionTreeRegressor(), 603.2217169624653), (DecisionTreeRegressor(splitter='random'), 603.2373201789363), (LinearRegression(), 770.1094857338295)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 193, [(DecisionTreeRegressor(splitter='random'), 787.6851045770052), (DecisionTreeRegressor(), 788.3442224068157), (LinearRegression(), 973.4035291070657)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 260, [(DecisionTreeRegressor(splitter='random'), 501.9119856680774), (DecisionTreeRegressor(), 506.26237682281646), (LinearRegression(), 710.3267466368568)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 56, [(DecisionTreeRegressor(splitter='random'), 748.1834949316035), (DecisionTreeRegressor(), 804.5844637544474), (LinearRegression(), 866.4556066721598)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 79, [(DecisionTreeRegressor(splitter='random'), 565.3400560624382), (DecisionTreeRegressor(), 565.3773834321653), (LinearRegression(), 702.2417082312037)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 90, [(DecisionTreeRegressor(splitter='random'), 525.8207089806359), (DecisionTreeRegressor(), 525.8234170600869), (LinearRegression(), 678.7556323136571)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 162, [(DecisionTreeRegressor(splitter='random'), 580.669991599885), (DecisionTreeRegressor(), 580.7143914702891), (LinearRegression(), 720.7364654244893)]), ('s3://air-example-data/ursa-labs-taxi-data/by_year/2019/05/data.parquet/359c21b3e28f40328e68cf66f7ba40e2_000000.parquet', 50, [(DecisionTreeRegressor(splitter='random'), 525.3179619518305), (DecisionTreeRegressor(), 525.3690225488036), (LinearRegression(), 672.5664989074589)])]\n"
      ]
     }
    ],
    "source": [
     "from sklearn.tree import DecisionTreeRegressor\n",
     "\n",
-    "results = run(\n",
+    "results = run_batch_training(\n",
```

Comment:
Should we include a section describing what to do with the results of this training job?